api/v1beta1/registryconfig_types.go

```diff
@@ -142,11 +155,46 @@ type RegistryCacheConfigStatus struct {
 
 	// List of status conditions to indicate the status of a ServiceInstance.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
-
-	// ProvisioningCompleted indicates if the initial provisioning of the cluster is completed
-	ProvisioningCompleted bool `json:"provisioningCompleted,omitempty"`
 }
 
 func init() {
 	SchemeBuilder.Register(&RegistryCacheConfig{}, &RegistryCacheConfigList{})
 }
+
+func (rc *RegistryCacheConfig) UpdateStatusPending(conditionType ConditionType, reason ConditionReason, status metav1.ConditionStatus) {
```

Comment:
Maybe we should simplify this interface - make `UpdateStatuPending` private and add more specialised function e.g. `RegistryCacheValidatedUpdatePendingTrue`  that would only have the `reason` arg?

Reply:
Done